pkgs/buildkite/cli/registry.yaml

```diff
@@ -0,0 +1,19 @@
+packages:
+  - type: github_release
+    repo_owner: buildkite
+    repo_name: cli
+    description: "A command line interface for Buildkite."
```

Comment:
```suggestion
    description: "A command line interface for Buildkite"
```

https://github.com/aquaproj/aqua-registry/blob/main/CONTRIBUTING.md#remove-a-period-from-the-end-of-the-description

> Remove a period from the end of the description